bzt/modules/blazemeter.py

```diff
@@ -226,10 +228,16 @@ def post_process(self):
         finally:
             self._postproc_phase2()
 
-        if self.client.results_url:
+        report_link = None
+        if self.client.token and self.public_report:
+            report_link = self.client.make_report_public()
```

Comment:
Good idea.

Initially i did just that. But then reconsidered, because two links printed together looked kind of boring. But automation is pretty important, so let's have two links.

Updated the PR.